jenkins/ci.suse.de/cloud-mkphyscloud-qa-test-upgrade-ui.yaml

```diff
@@ -140,7 +142,7 @@
               onadmin_upgrade_ses_to_4
             fi
 
-            export cloudsource=$cloudsource
+            export cloudsource=$original_cloudsource
```

Comment:
Ah, true. But if you need it for `onadmin_prepare_cloudupgrade_nodes_repos_6_to_7` only, maybe you can move that function above the ceph upgrade?